example/session/src/main/java/org/apache/iotdb/SessionExample.java

```diff
@@ -247,15 +247,11 @@ private static void insertTablet() throws IoTDBConnectionException, StatementExe
 
     Tablet tablet = new Tablet("root.sg1.d1", schemaList, 100);
 
-    long[] timestamps = tablet.timestamps;
```

Comment:
You should not delete the previous example, user also can use the previous way to add data.  You should add new code to show how to use.

Reply:
copy that,thank you